libs/libxx/0001-uclibxx-use-overload-constructor-of-filebuf-ostream.patch

```diff
@@ -0,0 +1,308 @@
+From b9028c8bc1a4cd00feb5328521bdde332fefeda3 Mon Sep 17 00:00:00 2001
+From: zhuyanlin <zhuyanlin1@xiaomi.com>
+Date: Mon, 27 Sep 2021 21:47:41 +0800
+Subject: [PATCH] uclibxx: use overload constructor of filebuf & ostream
+
+Instead of set valiable in Init, use overload constructor in
+filebuf & stream class
+
+Change-Id: I090432b5576eb568f92f3c147b39ad2efd8ba6b0
+---
+ include/fstream   | 15 +++++++--
+ include/ios       | 21 +++---------
+ include/istream   |  4 +++
+ include/ostream   |  8 +++++
+ include/streambuf |  6 ++--
+ src/ios.cpp       | 84 ++++++++---------------------------------------
+ 6 files changed, 44 insertions(+), 94 deletions(-)
+
+diff --git a/uClibc++/include/fstream b/uClibc++/include/fstream
+index fddfa35..99a016e 100644
+--- a/uClibc++/include/fstream
++++ b/uClibc++/include/fstream
```

Comment:
Here is the prefix:
https://github.com/apache/incubator-nuttx/blob/master/libs/libxx/0001-uClibcxx-basic_definitions-fix-GCC-specific-definiti.patch#L16-L17